tests/aws/services/stepfunctions/v2/scenarios/test_base_scenarios.py

```diff
@@ -1121,6 +1121,35 @@ def test_map_state_tolerated_failure_percentage_path(
             exec_input,
         )
 
+    @markers.aws.validated
+    def test_map_state_label(
```

Comment:
Would be good to add in `aws.services.stepfunctions.v2.test_sfn_api_map_run.TestSnfApiMapRun` some api validation for state machine creation that have invalid `Label` fields (empty, with illegal characters, too long, incorrect type). The tests could check for raising an exception on creation and skip the validation of the error message as we do not have a static analyser yet. (see comment above about assertions on Label instantiation)

Reply:
Fixed in [df3f18d](https://github.com/localstack/localstack/pull/11073/commits/df3f18ddb008ee9572898155f82643f608046ef7).